### ISSUE TITLE
Support PyPy 3.8

### ISSUE THREAD

**Issue by rodrigodc** [python-poetry/poetry#4655]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: CentOS 7.9
- **Poetry version**: 1.1.11
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: https://gist.github.com/rodrigodc/39f71913dacf213afa5e24e8190dbe2c

## Issue

Please update `virtualenv` when https://github.com/pypa/virtualenv/pull/2206 gets merged.

Reason: virtualenvs created by Poetry are broken (not isolated) on PyPy 3.8, as described at https://github.com/pypa/virtualenv/issues/2182.

A `python3.8` _dir_ is created in the venv (but remains empty, nothing is installed there) :point_down: 

```
root@15e2c53df66a /tmp/test # ls <venv dir>/lib/python3.8/site-packages/
```

A `pypy3.8` _symlink_ is created _pointing to the system (global) site-packages dir_ (and everything is installed there) :point_down: 

```
root@15e2c53df66a /tmp/test # ls -latr <venv dir>/lib/pypy3.8
lrwxrwxrwx. 1 root root 28 Oct 18 18:12 <venv dir>/lib/pypy3.8 -> /usr/lib64/pypy3/lib/pypy3.8
```

The mentioned PR fixes that.

I can keep track of it and let you know when it gets merged and a new `virtualenv` version released with the fix.

Thank you,
Rodrigo

**Comment by rodrigodc** [comment]
Would it be possible to release `1.1.12` with the fixed `virtualenv==20.9.0`, as described above?

**Comment by mkniewallner** [comment]
Hi, I believe this has since been fixed?

Even a fresh install of Poetry `1.1.11` will install `virtualenv==20.15.1`, in line with https://github.com/python-poetry/poetry/blob/1.1.11/pyproject.toml#L41
